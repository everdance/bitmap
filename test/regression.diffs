diff -U3 /home/xiaolei/Code/github/everdance/bitmap/test/expected/bitmap.out /home/xiaolei/Code/github/everdance/bitmap/test/results/bitmap.out
--- /home/xiaolei/Code/github/everdance/bitmap/test/expected/bitmap.out	2024-08-22 17:54:11.319361024 -0400
+++ /home/xiaolei/Code/github/everdance/bitmap/test/results/bitmap.out	2024-09-08 15:09:19.891331700 -0400
@@ -0,0 +1,10 @@
+CREATE TABLE tst (
+	i	int4,
+	t	text
+);
+INSERT INTO tst SELECT i%2, substr(md5(i::text), 1, 1) FROM generate_series(1,2000) i;
+CREATE INDEX bitmapidx ON tst USING bitmap (i);
+server closed the connection unexpectedly
+	This probably means the server terminated abnormally
+	before or while processing the request.
+connection to server was lost
