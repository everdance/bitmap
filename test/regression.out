# using temp instance on port 61699 with PID 2088235
not ok 1     - bitmap                                     16 ms
1..1
# 1 of 1 tests failed.
# The differences that caused some tests to fail can be viewed in the file "/home/xiaolei/Code/github/everdance/bitmap/test/regression.diffs".
# A copy of the test summary that you see above is saved in the file "/home/xiaolei/Code/github/everdance/bitmap/test/regression.out".
